ports/mpark-patterns/vcpkg.json

```
@@ -0,0 +1,17 @@
+{
+  "name": "mpark-patterns",
+  "version-date": "2019-10-03",
```

```suggestion
  "version-date": "2019-10-03",
```
Why update to master instead of `v0.3.0` https://github.com/mpark/patterns/releases/tag/v0.3.0 ?
It seems it has been a few years since the last maintenance.

---

I think [Workaround for GCC 8.1.](https://github.com/mpark/patterns/commit/23fbb3b64188e69137ef1ecff64da2dfdef9e340) [Fixed](https://github.com/mpark/patterns/commit/23fbb3b64188e69137ef1ecff64da2dfdef9e340) https://github.com/mpark/patterns/issues/9[.](https://github.com/mpark/patterns/commit/23fbb3b64188e69137ef1ecff64da2dfdef9e340)  and [Fixed linker error by defining a function.](https://github.com/mpark/patterns/commit/bd4b9b82d1e45358eaf27015cd314c574f32a6d4) those two commits that on master has fix serious bug which after v0.3.0